jaxley/modules/base.py

```diff
@@ -1226,6 +1240,10 @@ def _vis(
         morph_plot_kwargs: Dict,
     ) -> Axes:
         branches_inds = view["branch_index"].to_numpy()
+
+        if type == "volume":
+            return plot_comps(self[:], dims, ax, **morph_plot_kwargs)
```

Comment:
do we need the `[:]`?

Reply:
Input arg needs to be View. So [:] takes care of that if it's already a view it doesn't matter